__tests__/mlflow.spec.ts

```diff
@@ -0,0 +1,356 @@
+  const options = {
+    ...DefaultProcessorOptions,
+    mlflow: true
+  };
+  const issues = [
+    generateIssue({
+      options,
+      updatedAt: getDaysAgoTimestamp(15),
+      createdAt: getDaysAgoTimestamp(15),
+      assignees: [MAINTAINER],
+      user: {
+        login: NON_MAINTAINER,
+        type: 'User'
+      }
+    })
+  ];
+  const processor = createIssueProcessorMock({
+    options,
+    getIssues: createGetIssues(issues),
+    listIssueComments: async () => [
+      {
+        user: {
+          login: MAINTAINER,
+          type: 'User'
+        },
+        body: 'comment',
+        created_at: getDaysAgoTimestamp(15)
+      }
+    ]
+  });
+  await processor.setMaintainers();
+  const createCommentSpy = jest.spyOn(processor, 'createComment');
+  await processor.processIssues(1);
+
+  expect(createCommentSpy).toHaveBeenCalledTimes(1);
+  expect(createCommentSpy).toHaveBeenCalledWith(
+    expect.anything(),
+    expect.stringContaining('@non-maintainer Any updates here?')
+  );
+});
+
+test('Skip processing issue if last comment was posted by bot', async () => {
+  const options = {
+    ...DefaultProcessorOptions,
+    mlflow: true
+  };
+  const issues = [
+    generateIssue({
+      options,
+      updatedAt: getDaysAgoTimestamp(15),
+      createdAt: getDaysAgoTimestamp(15),
+      assignees: [MAINTAINER]
+    })
+  ];
+  const processor = createIssueProcessorMock({
+    options,
+    getIssues: createGetIssues(issues),
+    listIssueComments: async () => [
+      {
+        user: {
+          login: 'bot',
+          type: 'Bot'
+        },
+        body: 'comment',
+        created_at: getDaysAgoTimestamp(15)
+      }
+    ]
+  });
+  await processor.setMaintainers();
+  const createCommentSpy = jest.spyOn(processor, 'createComment');
+  await processor.processIssues(1);
+
+  expect(createCommentSpy).not.toHaveBeenCalled();
+});
+
+test('Ignore issue that is triaged and has has-closing-pr label', async () => {
+  const options = {
+    ...DefaultProcessorOptions,
+    mlflow: true
+  };
+  const issues = [
+    generateIssue({
+      options,
+      updatedAt: getDaysAgoTimestamp(15),
+      createdAt: getDaysAgoTimestamp(15),
+      labels: ['has-closing-pr'],
+      assignees: ['maintainer']
+    })
+  ];
+
+  const processor = createIssueProcessorMock({
+    options,
+    getIssues: createGetIssues(issues),
+    listIssueComments: async () => [
+      {
+        user: {
+          login: MAINTAINER,
+          type: 'User'
+        },
+        body: 'Thanks for filing the PR!',
+        created_at: getDaysAgoTimestamp(15)
+      }
+    ]
+  });
+  await processor.setMaintainers();
+  const createCommentSpy = jest.spyOn(processor, 'createComment');
+  await processor.processIssues(1);
+
+  expect(createCommentSpy).not.toHaveBeenCalled();
+});
+
+test('Ignore issue if it is triaged, has has-closing-pr label, and last comment is from non-maintainer', async () => {
```

Comment:
@dbczumar Test to address https://github.com/harupy/stale/pull/1/files#r911522494